docker/compose/hadoop.env

```diff
@@ -21,6 +21,15 @@ HIVE_SITE_CONF_javax_jdo_option_ConnectionUserName=hive
 HIVE_SITE_CONF_javax_jdo_option_ConnectionPassword=hive
 HIVE_SITE_CONF_datanucleus_autoCreateSchema=false
 HIVE_SITE_CONF_hive_metastore_uris=thrift://hivemetastore:9083
+HIVE_SITE_CONF_hive_metastore_uri_resolver=org.apache.hudi.hadoop.hive.NoOpMetastoreUriResolverHook
+HIVE_SITE_CONF_hive_metastore_event_db_notification_api_auth=false
+HIVE_SITE_CONF_hive_execution_engine=mr
```

Comment:
MapReduce is no longer supported in Hive 3.  Should this be configured with `tez`?

Reply:
So there is I think a cavaet to this, I believe that Mapreduce is supported in some areas of hive based on the hive code but in general the default execution engine for hive3 is tez. From integtest stand point it seems that its been working fine with this setting above but i can try changing to tez to see if it causes issues.